tiri/src/lib.rs

```diff
@@ -13,9 +13,21 @@
 //! straight line code for one path through a SIR flow-graph and (most) conventional control flow
 //! has been replaced by guards (calls can still appear in traces).
 //!
-//! The core structures of the interpreter are borrowed from Miri (the interpreter used in rustc to
-//! evaluate compile-time constant code). Where a Tiri struct has a direct Miri equivalent, a
-//! comment will point this out. You can find the Miri structs in src/librustc_mir/interpret/.
+//! The interpreter takes as input a TIR trace and repeatedly evaluates TIR statements at the index
+//! indiciated by the program counter (initially zero). Each statement will update the program
+//! counter accordingly.
+//!
+//! Program state is mutated through assignments to local variables, each of which is an instance
+//! of `Value`. Memory locations are mutated via the `Value::Indirect` variant.
+//!
+//! When a guard is encountered, the guard condition must evaluate true for interpretation to
```

Comment:
I think this para is redundant given that the para below says the same (minus the bit about aborting).

Reply:
OK